content/gettingstarted/linux.md

```diff
@@ -8,46 +8,93 @@ disableNextPrev: true
 
 There are many Linux distributions in existence and we strive to support as many as possible. 
 
+_If you don't see your distro in the lists bellow you have two options. Either open a ticket asking for support or if you feel adventurus follow the [guide how to add support for your linux distro]({{}}) and also consider making a PR so the community  can benefit._
```

Comment:
Do you think this would be better if we shifted it below the Community supported distro list? I think it reads better that way?